cmake/module/FindQRencode.cmake

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2024-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+#[=======================================================================[
+FindQRencode
+------------
+
+Finds the QRencode header and library.
+
+This is a wrapper around find_package()/pkg_check_modules() commands that:
+ - facilitates searching in various build environments
```

Comment:
> various build environments

Given that all the new code is just to facilitate MSVC/`vcpkg`, you could probably be a bit more concrete with the documentation. Otherwise someone might wonder why all of this is being done, when
```bash
  find_package(PkgConfig REQUIRED)
  pkg_check_modules(PC_QRencode QUIET libqrencode)
```
would otherwise work.

Reply:
The branch and the PR description have been updated.